tests/integration/views/test_request.py

```diff
@@ -111,6 +113,62 @@ def test_request_view_with_authored_request_file(airlock_client):
     assert "Remove this file" in response.rendered_content
 
 
+def test_request_view_with_submitted_file(airlock_client):
+    airlock_client.login(output_checker=True)
+    release_request = factories.create_release_request(
+        "workspace",
+        status=RequestStatus.SUBMITTED,
+    )
+    factories.write_request_file(release_request, "group", "file.txt", "foobar")
+    response = airlock_client.get(
+        f"/requests/view/{release_request.id}/group/file.txt", follow=True
+    )
+    assert "Remove this file" not in response.rendered_content
+    assert "Approve File" in response.rendered_content
+    assert "Reject File" in response.rendered_content
+    assert '    id="file-approve-button" type="submit"' in response.rendered_content
+    assert '    id="file-reject-button" type="submit"' in response.rendered_content
```

Comment:
Probably not worth checking for the ids here - the big functional test in `test_e2e.py` will need to use them though

Reply:
In fact the functional test is the best place to check that the right buttons show up and are clickable or not where you expect them to be

Reply:
👍 removed from here & added to `e2e.py`